RELEASING.md

```diff
@@ -0,0 +1,51 @@
+# Releasing OpenTelemetry Rust
+
+The primary audience for this is the SIG Maintaners. It provides the list of steps for how to release the crates and the
+considerations to make before releasing the create. It may provide use to consumers of the crate if/when we develop a
+release cadence.
+
+## Release cadence
+
+As of Feb 2024, there is no established cadence for the OpenTelemetry crates. The balance is required between too many
+breaking changes in a single release, and since we have instability flipping between implementations across 0.x
+releases. Perhaps once we've established a 1.0 we can consider a consistent release cadence.
```

Comment:
```suggestion
releases. Perhaps once we've established a 1.0 we can consider adopting a consistent release cadence.
```

Reply:
Note that for a bunch of these crates, semver stability might be limited by dependencies that appear in the public API.